src/cpp/session/modules/SessionCodeSearch.cpp

```diff
@@ -1399,13 +1401,16 @@ class SourceItem
 public:
    enum Type
    {
-      None = 0,
-      Function = 1,
-      Method = 2,
-      Class = 3,
-      Enum = 4,
+      None      = 0,
+      Function  = 1,
+      Method    = 2,
+      Class     = 3,
+      Enum      = 4,
       EnumValue = 5,
-      Namespace = 6
+      Namespace = 6,
+      Section   = 7,
```

Comment:
There's also equations and theorems (although the latter obscure enough that we probably don't need navigation for it).